flow-typed/overrides/Window.js

```diff
@@ -3,25 +3,15 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 // @flow
 
-import type { IDBFactory } from '../../src/types/indexeddb';
-
-type Lib = { debugName: string };
-
-declare class SymbolTable {
-  constructor(): SymbolTable,
-  getFuncAddressTableForLib(lib: Lib): Promise<Uint32Array>,
-  getSymbolsForAddressesInLib(
-    requestedAddressesIndices: number[],
-    lib: Lib
-  ): Promise<string[]>,
-}
+import type { IDBFactory, IDBKeyRange } from '../../src/types/indexeddb';
+import type { SymbolTableAsTuple } from '../../src/profile-logic/symbol-store-db';
 
 declare class GeckoProfiler {
   getProfile: () => Object,
   getSymbolTable: (
     debugName: string,
     breakpadId: string
-  ) => Promise<SymbolTable>,
+  ) => Promise<SymbolTableAsTuple>,
```

Comment:
oops :)

Note: maybe we should move this type, this doesn't seem appropriate to import it from `src/profile-logic/symbol-store-db`. Any suggestion ?

Reply:
I think the window definition mixes our metaphors between consuming code and our code a bit. It's a little weird, but it works. Do you have any other suggestions?

Reply:
I don't see anything better for now, but maybe we can at least declare our stuff in separate files. I'll look at this in a separate PR.